### ISSUE TITLE
BLE Journal

### ISSUE THREAD

**Issue by learning277** [Semper-Viventem/MetaRadar#142]
The journal on the ble radar is not working. 
This a pixel 3xl version 14

**Comment by lu-ckystars** [comment]
i have also noticed this issue on a pixel 5a

**Comment by Semper-Viventem** [comment]
Hey @cactushydrocodone , does it work if you wait long enough? I suspect it might be an issue with the loading stuff, can you see the progress bar displayed?

**Comment by Semper-Viventem** [comment]
Yeah, it might not be very clear. The journal is needed to keep the history of radar detections and reported system errors. I think some description is needed there

**Comment by Semper-Viventem** [comment]
> I might need to add: Ive installed the apk from github, as the play store says the app is not compatible with my device, whatever that means

Could you share some details about your device like the model, android version etc? I need to check the Google Play page configuration, if the apk runs on your device, Google Play should not limit it by itself.

**Comment by Semper-Viventem** [comment]
Also, I'd recommend using release versions instead of debug ones, they are more stable and consume less memory.

**Comment by Chiramisudo** [comment]
> Also, I'd recommend using release versions instead of debug ones, they are more stable and consume less memory. 

As for me, here's my version and device info, asking with the Exception I see on the Journal page with all permissions granted. 

**App Info**
Version: 0.32.1-beta (1708536381)
Build type: Release
Distribution: F-Droid

**System**
Android 16 on Pixel 7 Pro
Build BP4A.251205.006

FWIW, given Google's stupid new policy to block sideloading, I'm evaluating other ROMs that work on my phone but they must support apps like Microsoft Authenticator. My goal is to migrate off of Android for good. It was much easier to move to Linux from Windows. 

In any case, as a privacy advocate, I really like this project and would love to contribute if that's welcome. I'm an experienced and meticulous software engineer, but none with Kotlin, though it's been one of my goals to learn.

**Exception**:
![Screenshot_20260118-180641.png](https://github.com/user-attachments/assets/dca4f4e8-0a03-48ae-8a29-f3be8cdec130)
